tests/test_api.py

```diff
@@ -55,3 +55,50 @@ async def test_wrong_api_response():
     api = cro.ApiProvider(auth_required=False)
     with pytest.raises(cro.ApiError):
         await api.post('account')
+
+
+@pytest.mark.asyncio
+async def test_api_rate_limits():
+    api = cro.ApiProvider(from_env=True)
+    pair = cro.pairs.CRO_USDT
+
+    page = 0
+    page_size = 50
+
+    params = {'page_size': page_size, 'page': page}
+
+    if pair:
+        params['instrument_name'] = pair.name
+
+    start_time = time.time()
+    await api.post('private/get-order-history', {'params': params})
+    await api.post('private/get-order-history', {'params': params})
+    finish_time = (time.time() - start_time)
+
+    assert finish_time > 1
+
+    start_time = time.time()
+    await api.post('private/get-order-history', {'params': params})
+    await api.post('private/get-order-history', {'params': params})
+    await api.post('private/get-order-history', {'params': params})
+    await api.post('private/get-order-history', {'params': params})
+
+    finish_time = time.time() - start_time
+    assert finish_time > 4
+
+    start_time = time.time()
+    await api.get('public/get-instruments')
```

Comment:
you check here in synchronus way, so request limit in not working here, you should use "asyncio.gather" instead to request in parallel

Reply:
make 100 requests for example to check we not received rate limit error